config/nginx/dice-staging.tripleawarclub.org

```diff
@@ -0,0 +1,19 @@
+server {
+  listen 80;
+  listen [::]:80;
+  server_name dice-staging.tripleawarclub.org;
+  root /usr/share/nginx/html/dice-staging.tripleawarclub.org/public_html;
+
+  access_log /usr/share/nginx/html/dice-staging.tripleawarclub.org/logs/access.log;
+  error_log /usr/share/nginx/html/dice-staging.tripleawarclub.org/logs/error.log;
```

Comment:
👍 For using a fresh directory
What to do with the logs can be decided at another time.